ui/tests/integration/components/pki/pki-tidy-form-test.js

```diff
@@ -231,28 +274,23 @@ module('Integration | Component | pki tidy form', function (hooks) {
     assert.false(this.autoTidy.tidyAcme, 'tidyAcme is false on model');
 
     await click(PKI_TIDY_FORM.toggleInput('acmeAccountSafetyBuffer'));
-    await fillIn(PKI_TIDY_FORM.acmeAccountSafetyBuffer, 3); // units are days based on defaultValue
+    await fillIn(PKI_TIDY_FORM.acmeAccountSafetyBuffer, 2); // units are days based on defaultValue
```

Comment:
changed because 3 days was too similar to the default (72h vs 720h) and made it seem like there was a ttl miscalculation